config/crd/bases/run.tanzu.vmware.com_tanzukubernetesreleases.yaml

```diff
@@ -544,7 +544,7 @@ spec:
             type: object
         type: object
     served: true
-    storage: false
+    storage: true
```

Comment:
@imikushin I was not able to create/update spec for TKR ```v1alpha3``` object and @shyaamsn figured out this was the cause. Was it an intentional choice to leave ```storage: true``` for ```v1alpha1``` TKR?

Reply:
Changed to load a local tkr CRD with the above booleans switched and that fixed the problem